ReactNativeClient/lib/joplin-renderer/MdToHtml/copyFunction.js

```diff
@@ -0,0 +1,35 @@
+(function(btn) {
+	const MDIC = {
```

Comment:
Please use full words for function names and variables. `btn` = `button`. `MDIC` I don't know what it means.